index.js

```diff
@@ -1,5 +1,94 @@
 // Calculator project
+
+//Objects ///////////////////////////////////////////////////////////////
 const display = document.querySelector('.display');
-const one = document.querySelector('.one');
+const numberButtons = Array.from(document.getElementsByClassName('numberButton'));
```

Comment:
if you use `document.querySelector('.className')` then you'll automatically create an array of the elements with the same class name